projects/network_tester/inc/supervisor.h

```diff
@@ -21,6 +21,12 @@ typedef struct
 
 /*********************************************************************/
 /*****PUBLIC DEFS*****************************************************/
+// downlink band configs
+extern const llabs_dl_band_cfg_t DL_BAN_FCC;  // USA / Mexico
+extern const llabs_dl_band_cfg_t DL_BAN_BRA;  // Brazil
+extern const llabs_dl_band_cfg_t DL_BAN_AUS;  // Australia
+extern const llabs_dl_band_cfg_t DL_BAN_NZL;  // New Zealand
+extern const llabs_dl_band_cfg_t DL_BAN_ETSI; // Europe
```

Comment:
Any file that includes supervisor.h can view the dl_band_cfg for the following localization settings. This might be worth moving to the ifc at some point.